ruby/vampires.rb

```diff
@@ -0,0 +1,64 @@
+puts "How many employees will be processed?"
+employees = gets.chomp.to_i
+
+until employees == 0
+
+	puts "What is your name?"
+	name = gets.chomp
+
+	puts "How old are you?"
+	age = gets.chomp.to_i
+
+	puts "What year were you born?"
+	year_born = gets.chomp.to_i
+
+	puts "Our company cafeteria serves garlic bread. Should we order some for you (y/n)?"
+	garlic_bread = gets.chomp
+
+	puts "Would you like to enroll in the company's health insurance (y/n)?"
+	insurance = gets.chomp
+
+	puts "What allergies do you have (type 'done' when finished)?"
+	allergies = gets.chomp
+
+	real_age = 2017 - year_born
+
+	if age == real_age
+		age_is_correct = true
+	else
+		age_is_correct = false
+	end
+
+	allergies_done = false
+
+	until allergies_done 
+		if allergies == "sunshine"
+			allergies_done = true
+		elsif allergies == "done"
+			allergies_done = true
+		else 
+			allergies = gets.chomp
+		end
+	end
+
+	if allergies == "sunshine"
+		result = "Probably a vampire"
+	elsif age_is_correct && (garlic_bread == "y" || insurance == "y")
```

Comment:
You changed `age_is_correct` to a boolean, probably because there were two variables and it's easier to evaluate as one variable, but you can do that with garlic bread and insurance too. A long form would look similar to this:
```
if garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
    garlic_bread = true
else
   garlic_bread = false
end
```
can be refactored to:
```
garlic_bread = garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
```
If `garlic_bread.downcase` is neither of those answers, `garlic_bread` will be set to false.